### DISCUSSION TITLE
erros show me when i publish apk file with anes uisng adobe animate

### DISCUSSION THREAD

**Discussion by atiani** [airsdk/Adobe-Runtime-Support#1033] [Q&A]
recently i am uisng adobe animate to create android app and I used anes like advert anes and other  related anes but the big problem  is during publishing apk file the adobe animte show me some errors like this img in attachment below , so please any developers can help me to correctly this errors 
![ee](https://user-images.githubusercontent.com/78841476/127914754-2df9d184-4553-4cde-be87-147f2e481662.png)

best regards

**Comment by ajwfrost** [comment]
This is being covered by #1044 ... I think it may be a bug in the way ADT is formatting a string for multidex handling..
